packages/generator/src/shared/wireObjectTypeV2ToSdkObjectConst.ts

```diff
@@ -15,8 +15,14 @@
  */
 
 import type { ObjectTypeFullMetadata } from "@osdk/gateway/types";
+import { deleteUndefineds } from "../util/deleteUndefineds";
+import { stringify } from "../util/stringify";
 import { wireObjectTypeV2ToSdkObjectDefinition } from "./wireObjectTypeV2ToSdkObjectDefinition";
 
+export function getObjectDefIdentifier(name: string, v2: boolean) {
+  return v2 ? name : `${name}Def`;
```

Comment:
Is this backwards?

Reply:
Nope. In v1 we generate a type with the name `Foo` for you to use so we can't use that for the definition interface as well.